examples/discovery-search-app/scripts/setSdkUrl.js

```diff
@@ -29,50 +28,60 @@ function writeEnvVarToFileAndReturn(key, value) {
   return value;
 }
 
-module.exports = async function() {
+module.exports = async function () {
   validateSdkEnvVars();
 
   if (process.env.DISCOVERY_AUTH_TYPE === 'cp4d' && !process.env.DISCOVERY_URL) {
     const authUrl = process.env.DISCOVERY_AUTH_URL;
     const rejectUnauthorized = process.env.DISCOVERY_AUTH_DISABLE_SSL !== 'true';
-    const httpsAgent = new https.Agent({
-      rejectUnauthorized
-    });
+
+    let baseUrl = authUrl;
+    if (authUrl.endsWith('/icp4d-api')) {
+      baseUrl = authUrl.substring(0, authUrl.length - '/icp4d-api'.length);
+    }
+
+    let globalRejectUnauthorized = process.env.NODE_TLS_REJECT_UNAUTHORIZED;
+    if (!rejectUnauthorized) {
+      process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+    }
```

Comment:
For some reason, the previous code of creating an HTTPS agent with option `rejectUnauthorized: false` does not work with `axios.post()`. Only solution I found was to set `NODE_TLS_REJECT_UNAUTHORIZED`.

Reply:
LGTM doesn't like disabling certificate validation-- what about `rejectUnauthorized: false` doesn't work with `axios.post`?